aws-bill-pdf-to-csv/README.md

```diff
@@ -0,0 +1,50 @@
+This sample takes in a S3 URI pointing at a AWS Bill PDF and converts the bill into a CSV. Each item in the bill is converted to a row in the CSV. It will then write the CSV back to a provided S3 URI. This showcases how Griptape can be used to automate previously manual processes such as identifying if a PDF field with identical formatting is a `AWS Region` or an `AWS Service`.
```

Comment:
nit: may want to call out that somewhere in here or in the Requirements section that the AWS permissions provided should then have read/write access to both S3 URIs.

Reply:
Added links in **Configuration** section